deploy/kubernetes/deployment.yml

```diff
@@ -2,7 +2,7 @@ apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
   name: bookstore-broker
-  namespace: bookstore-broker
+  namespace: service-catalog
```

Comment:
Why the namespace change? That ripples thru a lot of files and doc references. The namespace `service-catalog` might be confused with the namespace that the actual Service Catalog extension is deployed to, or potentially clash with other service broker sample deployments. It would be better if the name were specific to `bookstore`.